Sources/Diagnostics/FileHandler.swift

```diff
@@ -74,12 +75,13 @@ actor FileHandler: FileHandlerType {
     }
 
     /// Adds a line at the end of the file
-    func append(line: String) {
+    @available(macOS 10.15.4, iOS 13.4, watchOS 6.2, tvOS 13.4, *)
+    func append(line: String) throws {
         RCTestAssertNotMainThread()
 
         self.fileHandle.seekToEndOfFile()
```

Comment:
There's also `seekToEnd()`, which is maybe newer? I genuinely don't know haha.

```suggestion
        self.fileHandle.seekToEnd()
```

Reply:
Oh, great catch! Thank you! Done in f16e0b96fd51f9f08105f7be941e726f428f0bce